.travis.yml

```diff
@@ -65,13 +64,15 @@ after_success:
 
 
 stages: #This sequence defines the order of the stages
-  - name: test
+  - name: Py311
+    if: branch = master OR branch = develop
+  - name: Py310
     if: branch = master OR branch = develop
   - name: lint
     if: branch = master OR branch = develop
   - name: deploy
     if: branch = master
-  - name: Py38
+  - name: Py38 #DEV MESSAGE: as long as we can pass the tests with Python 3.8, we will keep it. If a problem comes up, we will announce that RADIS is not compatible with Python 3.8 anymore.
```

Comment:
Seems like today is that day

Reply:
It's silly because it's only because the command `pd.set_option("future.no_silent_downcasting", True)` is not defined in this version of pandas. However, I'm not goign to implement a condition only for `python=3.8`. Well... bye bye Python 3.8